include/zephyr/ipc/pbuf.h

```diff
@@ -269,6 +269,20 @@ void pbuf_handshake_write(struct pbuf *pb, uint32_t value);
  */
 int pbuf_get_initial_buf(struct pbuf *pb, volatile char **buf, uint16_t *len);
 
+#if defined(CONFIG_ARCH_POSIX)
```

Comment:
don't guard the declaration in the header file, it's unnecessary

Reply:
This makes sense, thank you. It's anyway documented as an internal matter, so users don't call it